DEVELOPMENT.md

```diff
@@ -0,0 +1,62 @@
+# Development guide
+
+This doc describes how to bootstrap and run the project locally.
+
+## Setup
+
+### 1. Clone the repo.
+```shell
+git clone https://github.com/codex-team/codex.docs
+```
+
+### 2. Install dependencies
+
+```shell
+yarn install
+```
+
+### 3. Create separate config file for local overrides
+
+```shell
+touch app-config.local.yaml
+```
+
+### 4. Run the application
+
+```shell
+yarn dev
+```
+
+## Starting docs with MongoDB
```

Comment:
Describe that by default it will use local db

Reply:
done